### PR TITLE
Revise child, available-bundles, to bundles so it appears in navigation menus. 

### PR DESCRIPTION
**Pull request by mvincerx**
This PR changes the name of the file `available-bundles` to `bundles` so that it will appear as a menu option in the left-side nav on the clearlinux.org/documentation/clear-linux site. 

More explanation to follow via email.

### CODE REVIEW

File: source/clear-linux/concepts/bundles-about.rst [hunk 1]
```diff
@@ -17,15 +17,14 @@ individually or as a set, using tools such as "yum" and "apt-get."
 The |CLOSIA| takes a slightly different approach. While we also use the
```
Comment (krisdale) [inline source/clear-linux/concepts/bundles-about.rst]: Please update the CLOSIA substitution to CL-ATTR

File: source/clear-linux/concepts/security.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _security:
 
 Clear Linux\* OS Security
```
Comment (krisdale) [inline source/clear-linux/concepts/security.rst]: Please replace "Clear Linux\* OS" with CL-ATTR (title and first paragraph) :)

Comment (krisdale) [reply source/clear-linux/concepts/security.rst]: Also - under "Compiled with secure options" section, replace "Clear Linux" in text with CL substitution.

File: source/clear-linux/get-started/bare-metal-install/bare-metal-manual-install.rst [hunk 3]
```diff
@@ -1,10 +1,10 @@
 .. _bare-metal-manual-install:
 
-Install Clear Linux OS on bare metal (manual)
+Install |CL-ATTR| on bare metal (manual)
```
Comment (krisdale) [inline source/clear-linux/get-started/bare-metal-install/bare-metal-manual-install.rst]: 👍

File: source/clear-linux/get-started/virtual-machine-install/virtualbox.rst [hunk 4]
```diff
@@ -21,7 +21,7 @@ The instruction assumes that you have:
```
Comment (krisdale) [inline source/clear-linux/get-started/virtual-machine-install/virtualbox.rst]: Please update title/first paragraph to use CL-ATTR substitution. Also, under "Create a virtual machine in VirtualBox", replace "Clear Linux" in text with CL sub.

**Comment by krisdale** [comment]
@mvincerx Changes for the original purpose of this PR look good. With the addition of the substitution updates I described, we are good to go!

**Comment by mvincerx** [comment]
@krisdale . Thanks for your review.

**Comment by mltullis** [comment]
LGTM! Merging now.

**Pull request merged**